### DISCUSSION TITLE
Fire suppression CO2

### DISCUSSION THREAD

**Discussion by aiF1** [firemodels/fds#12958] [Q&A]
Hi. I am very new to FDS, SMV and Cfast. Tryig to find a code for fire suppression using CO2, instead of water, if readily available.  Gas is okay, foam is even better... 😁

**Comment by drjfloyd** [comment]
What are you looking to do? For system design, I would suggest you just follow the code design requirements. For example NFPA 12 provides fairly detailed requirements for designing a CO2 system.

**Reply by aiF1** [reply]
Hi. Thanks for reply. I am looking for a refence sample fds code or input commands for CO2 fire suppression.  Samples available are for water sprinklers. Is there any similar FDS sample sprinkler for CO2?

**Reply by drjfloyd** [reply]
There are no large scale CO2 validation cases. There are cup burner cases, but that configuration would not directly apply to a prototypical system.

Attempting to use FDS for CO2 suppression modeling would not be something for a brand new user to immediately jump into.  You will want to take some time gaining some FDS proficiency, make sure you understand how the FDS combustion and extinction models work, make you sure you understand the design principles for CO2 systems and the underlying physics and phenomena associated with the discharge of CO2 into a compartment.
